test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -433,7 +433,7 @@ public void TensorFlowTransformMNISTLRTrainingTest()
 
                 var pipe = mlContext.Transforms.Categorical.OneHotEncoding("OneHotLabel", "Label")
                     .Append(mlContext.Transforms.Normalize(new NormalizingEstimator.MinMaxColumn("Features", "Placeholder")))
-                    .Append(new TensorFlowEstimator(mlContext, new TensorFlowTransformer.Arguments()
+                    .Append(new TensorFlowEstimator(mlContext, new TensorFlowTransformer.Options()
```

Comment:
>TensorFlowEstimator [](start = 32, length = 19)

If you look on other peoples PR, i think we tend to change direct calls to estimators to mlContext.Transforms.TensorFlow